.circleci/config.yml

```diff
@@ -26,6 +27,41 @@ parameters:
     default: ""
 
 jobs:
+  test-go:
+    docker:
+      - image: circleci/golang:1.17
```

Comment:
should upgrade to the cimg/golang:1.17 image to avoid warnings about this in the UI.  doing this will also change the location of go packages to cache (think they are now in /home/circleci/go instead of /go).

Reply:
I've noticed that running docker in docker machine on circle is much faster than setup_remote_docker since remote docker uses a tiny separate instance (that's size is not configurable) Generally agree with Jon that these deps should be explicit and passed in. Here's an example of running postgres along side in circle https://circleci.com/docs/2.0/databases/